Firestore/core/src/local/leveldb_index_manager.h

```diff
@@ -52,6 +94,27 @@ class LevelDbIndexManager : public IndexManager {
    * be used to satisfy reads.
    */
   MemoryCollectionParentIndex collection_parents_cache_;
+
+  /**
+   * An in-memory map from collection group to a map of indexes associated with
+   * the collection groups.
+   *
+   * The nested map is an index_id to FieldIndex map.
+   */
+  std::unordered_map<std::string,
+                     std::unordered_map<int32_t, model::FieldIndex>>
+      memoized_indexes_;
+
+  QueueForNextIndexToUpdate next_index_to_update_;
+  int32_t memoized_max_index_id_ = -1;
+  int64_t memoized_max_sequence_number_ = -1;
+
+  /* Owned by LevelDbPersistence. */
+  LocalSerializer* serializer_ = nullptr;
+
+  bool started_ = false;
+
+  std::string uid_;
```

Comment:
This is not initialized, is it?

Reply:
It is? https://en.cppreference.com/w/cpp/string/basic_string/basic_string

Default constructor creates an empty string.

Reply:
Ack.